packages/global/styles/charts.scss

```diff
@@ -41,31 +41,31 @@ $tree-map-fill-default: #b2e6f6;
 $tree-map-fill-default-hover: #19C2F9;
 
 // pie chart
-$fill-pie-1: $fill-default;
-$fill-pie-2: $fill-secondary;
-$fill-pie-3: #6C6F72;
-$fill-pie-4: #c63663;
-$fill-pie-5: #00818b;
-$fill-pie-6: #c94000;
-$fill-pie-7: #007b4b;
-$fill-pie-8: #cb003b;
-$fill-pie-9: #006e91;
-$fill-pie-10: #9f0025;
-$fill-pie-11: #53565a;
-$fill-pie-12: #3b5e0f;
+$fill-1: $fill-default;
+$fill-2: $fill-secondary;
+$fill-3: #6C6F72;
+$fill-4: #c63663;
+$fill-5: #00818b;
+$fill-6: #c94000;
+$fill-7: #007b4b;
+$fill-8: #cb003b;
+$fill-9: #006e91;
+$fill-10: #9f0025;
+$fill-11: #53565a;
+$fill-12: #3b5e0f;
 
-$fill-pie-hover-1: #0087b3;
-$fill-pie-hover-2: #002747;
-$fill-pie-hover-3: #53565a;
-$fill-pie-hover-4: #b7274e;
-$fill-pie-hover-5: #006571;
-$fill-pie-hover-6: #d44600;
-$fill-pie-hover-7: #005e33;
-$fill-pie-hover-8: #aa0031;
-$fill-pie-hover-9: #00546f;
-$fill-pie-hover-10: #74001b;
-$fill-pie-hover-11: #45474b;
-$fill-pie-hover-12: #233809;
+$fill-hover-1: #0087b3;
+$fill-hover-2: #002747;
+$fill-hover-3: #53565a;
+$fill-hover-4: #b7274e;
+$fill-hover-5: #006571;
+$fill-hover-6: #d44600;
+$fill-hover-7: #005e33;
+$fill-hover-8: #aa0031;
+$fill-hover-9: #00546f;
+$fill-hover-10: #74001b;
+$fill-hover-11: #45474b;
+$fill-hover-12: #233809;
```

Comment:
Hey @hgaitanis not for now, but I think if in the future if we can make these scss vars to be able overridden, that will be helpful. The simple idea is add `!default` for each value so they are able to be overridden. We also may need to look at add prefix for each vars like `$myvic-fill-1`.

Reply:
Good points @tim-yao . Did not know about the `!default` option, seems like a handy feature